### ISSUE TITLE
AS3 SNATpool declaration: error happening after the first deployment

### ISSUE THREAD

**Issue by soubinan** [F5Networks/f5-appsvcs-extension#727]
### Environment
 * Application Services Version: 3.35
 * BIG-IP Version: 16.x

### Summary
I have a SNATPool declarared in my AS3 config, I have deployed it successfully on the first run
But now each time I deploy an updated version of the configuration with new object I receive a message error saying the SNAT IPs I declared previously are already existing.
And this error make my new load fail.
Note that I am not updating the SNATPool nor create anything related to it, just new vip and datagroup.
But it act like I was trying to create another SNATPool with the same IP address

### Steps To Reproduce
1. Submit the following declaration:
```json
"snatpool1": {
  "class": "SNAT_Pool",
  "remark": "snat pool 1",

  "snatAddresses": [
    10.10.10.1,
    10.10.10.2,
    10.10.10.3,
  ]
},
```

2. Observe the following error response: from restnoded.log
```json
{
  "message": "Declaration failed: 01020066:3: The requested SNAT Translation (/Common/Shared/10.10.10.1) already exists in partition Common.",
  "level": "error"
}
```

### Expected Behavior
On a new load the already existing objects should not be considered to be recreated.
They should be acknoledged and the lod should only take care of the new object to be loaded

### Actual Behavior
Each load is failling with a message saying the snatpool object is already exist.
AS3 seems to consider the snatpool declaration as a new creation each time they are encountered in the configuration, even when they are already loaded.

**Comment by adpuser98** [comment]
i experience the same issue/

Application Services Version: 3.45
BIG-IP Version: 16.1.3.2

**Comment by sunitharonan** [comment]
Hi @adpuser98,
AUTOTOOL-3851 has been created for the same and added to our product backlog. In order to prioritize, please reach out to us at automation_toolchain_pm@f5.com so we can prioritize.

**Comment by RobCupples** [comment]
@sunitharonan @soubinan @adpuser98 

This was fixed a couple of weeks ago with AUTOTOOL-3880 that originated from a Service Request.  The problem is limited to SNAT pools and/or SNAT translations created in /Common/Shared.  When I added the fix to the release notes I referred to this GitHub issue.  This fix will go out with our 3.46.0 release.
